src/amo/components/Link.js

```diff
@@ -18,7 +19,12 @@ export class LinkBase extends React.Component {
   }
 
   render() {
-    const { base, children, to } = this.props;
+    const { base, children, href, to } = this.props;
+
+    if (typeof href === 'string' && href.startsWith('/')) {
+      const linkHref = path.join(base, href);
+      return <Link {...this.props} href={linkHref}>{children}</Link>;
```

Comment:
Ah, I guess it's not documented but it will output a regular a tag with a href attribute that won't try to do any client-side routing. It basically outputs an <a> tag that isn't handled by React